docs/IJSValueWriter-api-windows.md

```diff
@@ -7,78 +7,78 @@ Kind: `interface`
 
 
 
-JSON-like stream writer.<br/>It is used to write data that is sent between native modules and the Microsoft.ReactNative.dll.<br/><br/>The JSON-like streams are data structures that satisfy the [JSON specification](https://tools.ietf.org/html/rfc8259). The data structure may have objects with name-value pairs and arrays of items. Property values or array items can be of type `Null`, `Object`, `Array`, `String`, `Boolean`, or `Number`. The `IJSValueWriter` treats the `Number` type as `Int64` or `Double`. See [`JSValueType`](JSValueType).<br/><br/>See the [`IJSValueReader`](IJSValueReader) for the corresponding reader interface.<br/><br/>The [`IJSValueReader`](IJSValueReader) and [`IJSValueWriter`](IJSValueWriter) must be rarely used directly. Use them to create serializer and deserializer functions for a type. The rest of application code must use these functions to serialize/deserialize values. The `Microsoft.ReactNative.Cxx` and `Microsoft.ReactNative.Managed` projects offer serializer/deserializer functions for many standard types. Use them directly or to define serializer/deserializer functions for your types.

```

Comment:
>JSON [](start = 0, length = 4)

This file was also generated based on the PR [https://github.com/microsoft/react-native-windows/pull/6891](https://github.com/microsoft/react-native-windows/pull/6891)

Reply:
Backporting it here: https://github.com/microsoft/react-native-windows/pull/7308